Jenkinsfile.integration

```diff
@@ -165,38 +181,48 @@ pipeline {
         }
 
         stage('Deploy Airship') {
+            when {
+                allof {
+                    expression { params.deployment == "airship" }
+                    expression { return FullDeploymentNeeded }
+                }
+            }
+
             options {
                 timeout(time: 45, unit: 'MINUTES', activity: true)
             }
-            when {
-                expression { params.deployment == "airship" }
-            }
             steps {
                 sh "./run.sh setup_airship"
             }
         }
     }
 
-    post {
-        failure {
-            script {
-                if (env.hold_instance_for_debug == 'true') {
-                    echo "You can reach this node by connecting to its floating IP as root user, with the default password of your image."
-                    timeout(time: 3, unit: 'HOURS') {
-                        input(message: "Waiting for input before deleting  env ${SOCOK8S_ENVNAME}.")
+    if (FullDeploymentNeeded) {
+        post {
+            failure {
+                when { expression { return FullDeploymentNeeded } }
```

Comment:
is this superflous as we are already inside the if?

Reply:
Argh. The "if" was supposed to go away. As it does not work. (See the failing tests)